pkg/autoscaler/autoscaler.go

```diff
@@ -262,21 +262,7 @@ var _ = Describe("[Feature:Machines] Autoscaler should", func() {
 		existingMachineSets, err := framework.GetMachineSets(client)
 		Expect(err).NotTo(HaveOccurred())
 		Expect(len(existingMachineSets)).To(BeNumerically(">=", 1))
-
-		By("Getting existing machines")
-		existingMachines, err := framework.GetMachines(client)
-		Expect(err).NotTo(HaveOccurred())
-		Expect(len(existingMachines)).To(BeNumerically(">=", 1))
```

Comment:
this seems like a sanity check, just to make sure we have machines. but i don't think it is directly related to the scale up/down functionality. also, `existingMachines` doesn't seem to be used elsewhere.

Reply:
Was this getting all machines from the cluster, or machines from a specific set?  It looks like it was all machines, in which case, we'd always have more than zero, so this can probably go.

Reply:
yeah, all the machinesets from openshift-machine-api project